datastore/filesystem/__init__.py

```diff
@@ -1,11 +1,7 @@
-__version__ = '1.1'
+__version__ = '1.2'
 __author__ = 'Juan Batiz-Benet <juan@benet.ai>'
```

Comment:
Do add yourself as an author!

Reply:
I'll do that once everything else is done 😃